README.md

```diff
@@ -31,8 +45,161 @@ $ docker-compose up
 
 After this, the server will be running at <http://localhost:8080>.
 
-## Enhancement ideas
+### Setting up the development environment
 
-- Add ability to sort key listing lexicographically in ascending or descending order.
-- Add ability to filter keys by some prefix query.
-- Add ability to delete keys.
+Create a development environment that is conducive for the candidate to edit
+and exercise their code. Candidates will likely have specific environments
+they prefer to work in, but for efficiency our environment will have common
+general-purpose tools. Ensure these are all in working order, and that the
+server is running before the interview starts.
+
+1. [Visual Studio Code](https://code.visualstudio.com/download) with Python
```

Comment:
I'll want to run this through it's paces a few times, but it might be nice to eventually have a scripted development environment that includes these things.